charts/selenium-grid/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: selenium-grid
 description: A Helm chart for creating a Selenium Grid Server in Kubernetes
 type: application
-version: 0.22.0
-appVersion: 4.13.0-20231004
+version: 0.22.1
```

Comment:
I was thinking about this again; this file cannot be changed before the release happens. Can you please revert this change and send it in a separate PR?